error-handling/src/main/java/com/palantir/remoting1/errors/SerializableError.java

```diff
@@ -49,7 +54,7 @@
 
     /** An optional representation of the server-side stacktrace of this error. */
     @Nullable
-    public abstract List<StackTraceElement> getStackTrace();
+    public abstract List<String> getStackTrace();
```

Comment:
that's problematic, though?

maybe we should make `List<StackTraceElement>` totally optional and support also a `getCallStack` or something?